cfpb/cfgov-refresh
# Buying-a-house: Update vulnerable OAH dependencies

Supersedes 
https://github.com/cfpb/cfgov-refresh/pull/5696/
https://github.com/cfpb/cfgov-refresh/pull/5694/
https://github.com/cfpb/cfgov-refresh/pull/5691/
https://github.com/cfpb/cfgov-refresh/pull/5657/

## Changes

- Update `jquery`.
- Update `handlebars`
- Update `axios`

## Testing

1. Pull branch, run `./frontend.sh`
2. Visit rate checker http://localhost:8000/owning-a-home/explore-rates/
3. Enter a house price of $1,000,000. The loan warning template should show (county-conf-warning.hbs) and the county drop-down should appear above it.
